### ISSUE TITLE
Handling large results files for publication

### ISSUE THREAD

**Issue by ghing** [openelections/openelections-core#160]
tl;dr:
- Per-election raw result CSVs are big (up to ~39M)
- Should we generate per-reporting-level files to reduce file size somewhat?
- Is a ~27M file (all precinct-level results for an election) too big for our target users?

As I've gotten into the implementation of publishing raw result files (#156), I've realized that the files get quite large, especially when we include Raw Results at all reporting levels in a single file.

The largest file is for the 2010 primary election and is ~39M for a file that includes all raw results for the county, precinct and state_legislative reporting levels.

For a file containing just the precinct-level results, the file is ~27M.

I'm worried that this might cause some issues with the GitHub API and it could possibly reach toward the [limits on repository size](https://help.github.com/articles/what-is-my-disk-quota).  I'd want to test this before I naysay using GitHub as the storage mechanism.

However, before getting into experimenting with committing such large files, I wanted to see if such large files were a problem for our target users.  Is a 27M file too large?  I'll note that I was able to open even the largest file without problem LibreOffice, but didn't try any filtering or processing.
